src/test/java/org/graylog/integrations/s3/codec/ApplicationJsonCodecTest.java

```diff
@@ -0,0 +1,93 @@
+package org.graylog.integrations.s3.codec;
+
+import org.graylog.integrations.s3.Configuration;
+import org.graylog2.gelfclient.GelfMessage;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import java.io.IOException;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ApplicationJsonCodecTest {
+
+    // Code Under Test
+    @InjectMocks
+    private ApplicationJsonCodec cut;
+
+    // Mocks
+    @Mock
+    Configuration mockConfig;
```

Comment:
I believe this unused field can be removed.
```suggestion
```

Reply:
Indeed it can.